src/encoded/static/components/publication.js

```diff
@@ -26,7 +28,64 @@ const PublicationComponent = (props, reactContext) => {
     ];
 
     const crumbsReleased = (context.status === 'released');
-
+    const accessions = context.datasets.reduce((dataset1, dataset2) => `${dataset1}&accession=${dataset2.accession}`, '');
+    const sort = (a, b) => a - b;
+    const datasetsColumns = {
+        accession: {
+            display: dataset => <a href={dataset['@id']}>{dataset.accession}</a>,
+            sort,
+            title: 'Accession',
+        },
+        type: {
+            getValue: datasets => (datasets['@type'] || [{}])[0], // only the first matters
+            sort,
+            title: 'Type',
+        },
+        target: {
+            getValue: dataset => (dataset.target || {}).label,
+            sort,
+            title: 'Target',
+        },
+        assay_term_name: {
+            getValue: (dataset) => {
+                // assay_term_name can be a string or an array
+                if (typeof (dataset.assay_term_name) === 'string') {
+                    return dataset.assay_term_name;
+                }
+                return (dataset.assay_term_name || []).join(', ');
+            }, 
```

Comment:
Getting a trailing space error in ESLint on this line.

Reply:
Removed.